src/fastcs/attributes.py

```diff
@@ -84,13 +108,16 @@ def updater(self) -> Updater | None:
 
 
 class AttrW(Attribute[T]):
+    """A write-only `Attribute`."""
+
     def __init__(
         self,
         datatype: DataType[T],
         access_mode=AttrMode.WRITE,
+        group: Optional[str] = None,
         handler: Sender | None = None,
     ) -> None:
-        super().__init__(datatype, access_mode, handler)  # type: ignore
+        super().__init__(datatype, access_mode, group, handler)  # type: ignore
```

Comment:
Perhaps covered by my comments on `handler` for Attribute / AttrR; don't go for inconsistent `__init__` signatures between multiple types in a class hierarchy like this. It is very brittle.